tensorflow/virtual_root_template_v1.__init__.py

```diff
@@ -1,133 +1,133 @@
-  def __init__(self, local_name, parent_module_globals, name):
-    self._local_name = local_name
-    self._parent_module_globals = parent_module_globals
-    super(_LazyLoader, self).__init__(name)
-
-  def _load(self):
-    """Import the target module and insert it into the parent's namespace."""
-    module = _importlib.import_module(self.__name__)
-    self._parent_module_globals[self._local_name] = module
-    self.__dict__.update(module.__dict__)
-    return module
-
-  def __getattr__(self, item):
-    module = self._load()
-    return getattr(module, item)
-
-  def __dir__(self):
-    module = self._load()
-    return dir(module)
-
-  def __reduce__(self):
-    return __import__, (self.__name__,)
-
-
-# Forwarding a module is as simple as lazy loading the module from the new path
-# and then registering it to sys.modules using the old path
-def _forward_module(old_name):
-  parts = old_name.split(".")
-  parts[0] = parts[0] + "_core"
-  local_name = parts[-1]
-  existing_name = ".".join(parts)
-  _module = _LazyLoader(local_name, globals(), existing_name)
-  return _sys.modules.setdefault(old_name, _module)
-
-
-# This list should contain all modules _immediately_ under tensorflow
-_top_level_modules = [
-    "tensorflow._api",
-    "tensorflow.python",
-    "tensorflow.tools",
-    "tensorflow.core",
-    "tensorflow.compiler",
-    "tensorflow.lite",
-    "tensorflow.keras",
-    "tensorflow.compat",
-    "tensorflow.summary",  # tensorboard
-    "tensorflow.examples",
-]
-# Estimator needs to be handled separatedly so we can still allow both
-# import tensorflow_estimator and import tensorflow.estimator work
-# Only in the second case do we actually need to do forwarding, the first case
-# already defines most of the hierarchy and eagerly forwarding would result in
-# an import loop.
-if "tensorflow_estimator" not in _sys.modules:
-  _root_estimator = False
-  _top_level_modules.append("tensorflow.estimator")
-else:
-  _root_estimator = True
-
-# Lazy load all of the _top_level_modules, we don't need their names anymore
-for _m in _top_level_modules:
-  _forward_module(_m)
-
-# We still need all the names that are toplevel on tensorflow_core
-from tensorflow_core import *
-
-_major_api_version = 1
-
-# In V1 API we need to print deprecation messages
-from tensorflow.python.util import deprecation_wrapper as _deprecation
-if not isinstance(_sys.modules[__name__], _deprecation.DeprecationWrapper):
-  _sys.modules[__name__] = _deprecation.DeprecationWrapper(
-      _sys.modules[__name__], "")
-
-# These should not be visible in the main tf module.
-try:
-  del core
-except NameError:
-  pass
-
-try:
-  del python
-except NameError:
-  pass
-
-try:
-  del compiler
-except NameError:
-  pass
-
-try:
-  del tools
-except NameError:
-  pass
-
-try:
-  del examples
-except NameError:
-  pass
-
-# LINT.ThenChange(//tensorflow/virtual_root_template_v2.__init__.py.oss)
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.

```

Comment:
Something is really wrong in how you've made this PR. The diff should not show the entire file as being changed